tests/complete/main.tf

```diff
@@ -63,7 +63,7 @@ module "eks" {
   version = "~> 19.10"
 
   cluster_name                   = local.name
-  cluster_version                = "1.26"
+  cluster_version                = "1.25"
```

Comment:
is there an issue with using 1.26?

Reply:
No support for aws-guardduty-agent on 1.26 yet.